R/tree_accessors.R

```diff
@@ -2213,6 +2226,19 @@ setMethod("subtitles<-", "VTitleFooter",
               obj
           })
 
+##' @rdname formatters_methods
+##' @export
+setMethod("subtitles", "TableRow",
+          function(obj) character())
+
+
+##' @rdname formatters_methods
+##' @export
+setMethod("subtitles<-", "TableRow",
+          function(obj, value) {
+              obj@subtitles <- value
```

Comment:
`TableRow` class does not have this slot. This function will break if it is ever called.

Same for all the other title/footer setters on `TableRow`

This also means our test coverage is insufficient